.golangci.yml

```diff
@@ -20,7 +20,13 @@ linters:
     - misspell
     # - staticcheck
     - unconvert
+    - typecheck
     # - unused
+    - staticcheck
+    - bidichk
+    - durationcheck
+    - exportloopref
+    - whitespace
```

Comment:
These cause plenty of [new lint failures](https://github.com/celo-org/celo-blockchain/actions/runs/6285414958/job/17067791540?pr=2189). So either we do not add these checks, or we fix the resulting failures.